thrall/app/lib/ElasticSearch.scala

```diff
@@ -70,10 +69,7 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
       andFilter(
         missingOrEmptyFilter("exports"),
         missingOrEmptyFilter(identifierField(Config.persistenceIdentifier)),
-        boolFilter.mustNot(termFilter(editsField("archived"), true)),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), StaffPhotographer.category)),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), ContractPhotographer.category)),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), CommissionedPhotographer.category))
+        boolFilter.mustNot(termFilter(editsField("archived"), true))
```

Comment:
I was wondering if we should also allow deleting archived images, and perhaps even images with a picdarUrn?